### DISCUSSION TITLE
A lot of CRC errors -- is this expected?

### DISCUSSION THREAD

**Discussion by bazmaroon** [omerfaruk-aran/esphome_samsung_hvac_bus#138] [General]
I'm seeing a lot of crc errors in the logs and want to know if that's expected.  Am I missing important messages?  Is there something that I should change to improve the connection?  I'm using an "M5Stack Atom Matrix ESP32 Development Kit" as my esp32 connection to my Samsung AC030KNZDCH/AA unit.  Attached is a log showing a minute or two of activity.
[logs_ac_logs.txt](https://github.com/lanwin/esphome_samsung_ac/files/14541499/logs_ac_logs.txt)

Thanks for what has been a fun activity getting my AC integrated into my Home Assistant.

**Comment by lanwin** [comment]
Hi @bazmaroon from what I currently know is that these CRC errors seems to be normal. My system also have them. I also checked the code from Samsung SNET and it also discards this messages as invalid and ignores them. You can safely raise up your log level so you dont see them.

> Thanks for what has been a fun activity getting my AC integrated into my Home Assistant.

You are welcome!
